modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -4513,6 +4513,16 @@
 		],
 		"to": "getCommerceInventoryBookedQuantity(param#0#, param#1#, StringPool.BLANK)"
 	},
+	{
+		"classNames": [
+			"CommerceShippingEngine"
+		],
+		"classStructurePattern": true,
+		"issueKey": "LPD-48918",
+		"newMethods": [
+			"@Override\n\tpublic String getKey() {\n\t\treturn KEY;\n\t}"
```

Comment:
@regiCesar21  what will happen if the client didn't name its variable `key`? It's mandatory?